src/main/java/hudson/plugins/jira/JiraChangeLogAnnotator.java

```diff
@@ -127,7 +151,7 @@ private void saveIssues(AbstractBuild<?, ?> build, JiraBuildAction a,
         }
     }
 
-    JiraSite getSiteForProject(AbstractProject<?, ?> project) {
+    JiraSite getSiteForProject(Job<?, ?> project) {
```

Comment:
I agree that using Item is not part of this change, but still good to learn about this, thanks :+1: